app/src/main/res/values/colors.xml

```diff
@@ -3,4 +3,5 @@
     <color name="colorPrimary">#008577</color>
     <color name="colorPrimaryDark">#980026</color>
     <color name="colorAccent">#D81B60</color>
+    <color name="lightGrey">#f2f2f2</color>
```

Comment:
Please remove this. There are no references to this color in the app.

Reply:
Done 👍